scripts/trigger-jenkins-build.js

```diff
@@ -0,0 +1,134 @@
+'use strict'
+
+const request = require('request')
+
+const githubClient = require('../lib/github-client')
+const botUsername = require('../lib/bot-username')
+
+const jenkinsApiCredentials = process.env.JENKINS_API_CREDENTIALS || ''
+
+function ifBotWasMentioned (commentBody, cb) {
+  botUsername.resolve((err, username) => {
+    if (err) {
+      return cb(err)
+    }
+
+    const atBotName = new RegExp('^@' + username)
```

Comment:
Why `^`? I could see myself including this in other comments as well (like the original PR description … unless I’m missing something?)